app/services/MemberDetailsUploadValidator.scala

```diff
@@ -164,9 +167,9 @@ class MemberDetailsUploadValidator @Inject()(
           case nameDOBFormProvider.firstName => ValidationErrorType.FirstName
           case nameDOBFormProvider.lastName => ValidationErrorType.LastName
           case nameDOBFormProvider.dateOfBirth => ValidationErrorType.DateOfBirth
-          case dobDayKey => ValidationErrorType.DateOfBirth
-          case dobMonthKey => ValidationErrorType.DateOfBirth
-          case dobYearKey => ValidationErrorType.DateOfBirth
+          case `dobDayKey` => ValidationErrorType.DateOfBirth
```

Comment:
As a matter of interest, what is the reason for this change?

Reply:
it makes it pattern match against the value of `dobDayKey` defined above instead of assigning the value to `dobDayKey` to be used in the pattern matching function